src/App.tsx

```diff
@@ -17,18 +37,22 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                todos={todos}
+                setFilter={setFilter}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {loading && (<Loader />)}
+              <TodoList filter={filter} setChosenTodo={setChosenTodo} handleToggleModal={handleToggleModal} chosenTodo={chosenTodo} />
```

Comment:
Show only one component, <Loader /> or <TodoList />

Reply:
Noted, but the code already meets this criteria. I think you might have commented on the outdated version.